BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -525,8 +524,11 @@ Cryptographic algorithms weaken over time as cryptanalysis improves: algorithms
                encryption (which provide 40 or 56 bits of security).
 
   Rationale: Based on {{!RFC3766}}, at least 112 bits 
-               of security is needed.  40-bit and 56-bit security are considered 
-               insecure today.  TLS 1.2 never negotiates 40-bit or 56-bit export ciphers and such ciphers are not supported at all in TLS 1.3.
+               of security is needed.  40-bit and 56-bit security (found in 
+               so-called "export ciphers") are considered 
+               insecure today.  Although such ciphers are allowed under TLS 1.2, 
```

Comment:
Not sure _"allowed"_ is the right word.  Export ciphers are already "MUST NOT negotiate" in TLS 1.1 (see https://www.rfc-editor.org/rfc/rfc4346#appendix-A.5), and have been kept alive only for when version negotiation selects 1.0 or earlier.